app/services/conversation.js

```diff
@@ -0,0 +1,65 @@
+import BaseServie from "./base.js";
+import User from "../models/user.js";
+
+export default class ConversationService extends BaseServie {
+  async #validateNewParticipantsIds(existingParticipantsIds, newParticipants) {
```

Comment:
`validate` term means that the function should return true/false, e.g. valid or invalid.
Maybe we could come up with a better name, e.g.

- `#validateAndReturnParticipantsIdsToAdd`
- `#validateAndReturnParticipantsIdsToRemove`